WooCommerce/WooCommerceTests/POS/Controllers/PointOfSaleOrderListControllerTests.swift

```diff
@@ -315,4 +316,84 @@ final class PointOfSaleOrderListControllerTests {
         #expect(orders == searchOrders)
         #expect(orderListService.lastSearchTerm == "test")
     }
+
+    @Test func updateOrder_when_order_loaded_from_API_then_order_list_updates() async throws {
+        // Given - load initial orders
+        let initialOrders = MockPointOfSaleOrderListService.makeInitialOrders()
+        orderListService.orderPages = [initialOrders]
+        await sut.loadOrders()
+
+        // Setup updated order
+        let orderToUpdate = initialOrders[0]
+        let updatedOrder = POSOrder(
```

Comment:
I'm not sure if we discussed this previously, or if wasn't worth it due to the small data structures for POS, but do you think we want to make POSOrders and others GeneratedFakeable/Copiable?So we would only need to do things like `POSOrder.fake().copy(customerEmail: "updated@example.com")`